Добавлены инструкции по запуску сервиса на Windows и Linux

timurash opened the pull request:
https://github.com/timurash/timeTrackingService/issues/16#issue-684467730

timurash committed:
Добавлены инструкции по запуску сервиса на Windows и Linux

https://github.com/timurash/timeTrackingService/issues/16#issue-684467730

timurash committed:
Пара правок в разделе установки сервиса на ОС Ubuntu

AlexandrYZ commented on Readme.md:
````diff
@@ -15,4 +15,194 @@
+```
+14. Проверим сосотяние сервиса командой:
+```
+sudo systemctl status timeTrackingService.service
````
Наличие примера вывода команды при успешном запуске сервиса сделает процесс диагностики проще.

timurash committed:
В файл Readme.md добавлен пример вывода команды systemctl status при успешном запуске сервиса

https://github.com/timurash/timeTrackingService/pull/18/commits/aa84ac0c704dd03260191211ad37ab98c4181e0a#r476747415
